pkgs/development/compilers/graalvm/graalvm-oracle/default.nix

```diff
@@ -3,8 +3,9 @@
   stdenv,
   fetchurl,
   graalvmPackages,
+  onnxruntime,
```

Comment:
I don't know if this is needed anymore, does `autoPatchelfIgnoreMissingDeps` even notice that this is in the environment? Or will the program fail at some point if it wantred  to use sth from `libonxxruntime.so`?

Reply:
```suggestion
```
that's unused

Reply:
Thanks @SuperSandro2000, we were both wondering about that!

Reply:
@farnoy Can you remove this and force-push?